mixer/pkg/config/crd/store.go

```diff
@@ -132,10 +137,17 @@ func (s *Store) checkAndCreateCaches(
 		if retryCtx.Err() != nil {
 			break
 		}
-		if retry {
-			log.Debug("Retrying to fetch config...")
+		if retryCount > 0 {
+			if retryCount%logPerRetries == 1 {
+				remainingKeys := make([]string, 0, len(kinds))
+				for k := range kindsSet {
+					remainingKeys = append(remainingKeys, k)
+				}
+				log.Debugf("Retrying to fetch config: %+v", remainingKeys)
```

Comment:
we should consider adding a LOG_EVERY_N like feature to our logger so code like this doesn't need to be manually rolled out every time

that or the syslog feature of just saying what repeats - wdty @geeknoid ?